README.md

```diff
@@ -37,51 +37,74 @@ Check out the [🌩 **Nimbus Documentation Hub**](https://experimenter.info) or
 1. Prerequisites
 
     On all platforms:
-    - Install [Docker](https://www.docker.com/)
-    - Install [Node](https://nodejs.org/en/) ^14.0.0
-
+    - Install [Node](https://nodejs.org/en/) ^14.0.0 <17.0 (recommended: 16.9.1)  
+    
     On Linux:
     - [Setup docker to run as non-root](https://docs.docker.com/engine/security/rootless/)
 
     On MacOS:
-      - [Install Docker Desktop](https://www.docker.com/products/docker-desktop)
-      - Adjust resource settings
-        - CPU: Max number of cores
-        - Memory: 50% of system memory
-        - Swap: Max 4gb
-        - Disk: 100gb+
-
-1.  Clone the repo
+      - Install [yarn](https://github.com/yarnpkg)
+
+            brew install yarn
+            
+      - Install [Docker](https://docs.docker.com/desktop/mac/install/)
+          - Adjust resource settings
+            - CPU: Max number of cores
+            - Memory: 50% of system memory
+            - Swap: Max 4gb
+            - Disk: 100gb+
+            
+1. Clone the repo
 
         git clone <your fork>
 
-1.  Copy the sample env file
+1. Copy the sample env file
 
         cp .env.sample .env
 
-1.  Set DEBUG=True for local development
+1. Set DEBUG=True for local development
 
         vi .env
 
-1.  Create a new secret key and put it in .env
+1. Create a new secret key and put it in .env
 
         make secretkey
 
-1.  Run tests
+	vi .env
+
+	```
+	...
+	SECRETKEY=mynewsecretkey
+	...
+	```
+
+1. Run tests
 
         make check
 
-1.  Setup the database
+1. Setup the database
 
         make refresh
 
+1. If using VSCode, configure workspace folders
+
+   - Add `app` folder to your workspace (File -> Add Folder to Workspace -> `path/to/experimenter/app`)
+   - From the `app` folder, run `yarn install`
+       - Make sure you are using the correct version of node
+          
+           node -v
+
+       - Troubleshooting: 
+           - [Changing node version](https://stackoverflow.com/a/50817276/12178648)
+           - Clear npm cache: `npm cache clean --force`
+
```

Comment:
A couple thoughts, feel free to take or leave...

Later in the README (step 2 under Semi Dockerized Setup Steps) is where we currently mention `yarn install`, would it be better to put this block there? Or at least everything under the second item about running `yarn install` and troubleshooting?
- If this is moved, then also don't need the `brew install yarn` for MacOS Prereqs
- If this is _not_ moved, we should add a link to [yarn's install guide](https://classic.yarnpkg.com/lang/en/docs/install/) to the Linux prereqs section

The VSCode workspace configuration could potentially be its own section if we want a spot to put general IDE setup tips.

Reply:
I haven't used the semi-dockerized instructions, so it might be better to pull this `yarn` stuff out into its own section. I was the same thing about the vscode info too 🤔